### ISSUE TITLE
Debian package: Spuriously installs files to `/pkg/`

### ISSUE THREAD

**Issue by maxb** [grafana/grafana#74002]
### What happened?

Install Grafana as per https://grafana.com/docs/grafana/latest/setup-grafana/installation/debian/

The package spuriously includes files in the top-level directory `/pkg/`:

```
# dpkg -L grafana | grep ^/pkg
/pkg
/pkg/etc
/pkg/etc/default
/pkg/etc/default/grafana-server
/pkg/etc/init.d
/pkg/etc/init.d/grafana-server
/pkg/usr
/pkg/usr/lib
/pkg/usr/lib/systemd
/pkg/usr/lib/systemd/system
/pkg/usr/lib/systemd/system/grafana-server.service
```

### What did you expect to happen?

Extra spurious files should not be installed

### Did this work before?

Yes

### How do we reproduce it?

As above

### Environment (with versions)?

Grafana: OSS 10.1.0

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

**Comment by raggi** [comment]
I checked older release debs, this appears to have been introduced with 10.0.4.

**Comment by kminehart** [comment]
Thanks for the report and the easy to reproduce steps. While it's not exactly a duplicate this was resolved by the same solution as #73870 (fix here: https://github.com/grafana/grafana-build/pull/141). I'll leave this open until we've fixed the packages in the repository.

**Comment by kminehart** [comment]
This is now resolved with the v10.1.1 release. Apologies for this one :sweat_smile: 

These don't return any results on the new package
```
dpkg -c grafana_10.1.1_armhf.deb | grep -i pkg
dpkg -c grafana_10.1.1_armhf.deb | grep -i tar.gz
```
